src/Expr.js

```diff
@@ -169,38 +214,113 @@ var exprToString = function(expr, caller) {
 
   if ('object' in expr) return printObject(expr['object'])
 
-  // Versioned queries/lambdas will have an api_version field.
-  // We want to prevent it from being parsed and displayed as:
-  // Query(ApiVersion("3", "X", Var("X")))
-  var keys = Object.keys(expr).filter(
-    expression => expression !== 'api_version'
-  )
-  var fn = keys[0]
+  if ('select' in expr) {
+    return (
+      'Select(' +
+      exprToString(expr['select']) +
+      ', ' +
+      exprToString(expr['from']) +
+      ')'
+    )
+  }
 
-  // For FQL functions with special formatting concerns, we
-  // use the specialCases object above to define their casing.
-  if (fn in specialCases) fn = specialCases[fn]
+  if ('containsstr' in expr) {
+    return (
+      'ContainsStr(' +
```

Comment:
are you planning to add other string functions here? `containsstr` is already handled by `specialCases` map, if you plan to add those functions here, you can remove from there

Reply:
@marrony I guess we could remove them from the special cases map and add all the `Contains` functions here since we need to make sure the argument order is constructed properly 👍 

What do you think?